applications/integrationtests/CMakeLists.txt

```diff
@@ -1,25 +1,14 @@
-if(ENABLE_GEOMETRYCONCRETE)
-    add_subdirectory(geometryConcrete)
-endif()
-
-if(ENABLE_METAMODEL)
-    add_subdirectory(metamodel)
-endif()
-
-if(ENABLE_OPTIMIZE)
-    add_subdirectory(optimize)
-endif()
+unset(all_integration_tests CACHE)
 
+add_subdirectory(geometryConcrete)
+add_subdirectory(math)
 add_subdirectory(mechanics)
-
-if(ENABLE_VISUALIZE)
-    add_subdirectory(visualize)
-endif()
-
-
-add_executable(SparseMatrixTests SparseMatrixTests.cpp)
-target_link_libraries(SparseMatrixTests Math Base)
-add_test(integration::SparseMatrixTests.cpp
-    ${CMAKE_CURRENT_BINARY_DIR}/SparseMatrixTests)
-
-add_custom_target(integrationtests COMMAND ctest -R integration)
+add_subdirectory(metamodel)
+add_subdirectory(optimize)
+add_subdirectory(visualize)
+
+add_custom_target(integrationtests
```

Comment:
Is it possible/reasonable to collect these target definitions somewhere? /srcipts/Targets.cmake or something?

Reply:
I guess it would be possible, but I think it's quite appropriate there. In the `integrationtests` directory, you define your `integrationtests` target. Like you would with the `src/mechanics` and `Mechanics`, no?